src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/OppfolgingstilfelleArbeidstakerSpek.kt

```diff
@@ -0,0 +1,190 @@
+package no.nav.syfo.oppfolgingstilfelle
+
+import no.nav.syfo.dialogmotekandidat.domain.DIALOGMOTEKANDIDAT_STOPPUNKT_DURATION_DAYS
+import no.nav.syfo.dialogmotekandidat.domain.DialogmotekandidatEndring
+import no.nav.syfo.testhelper.UserConstants.ARBEIDSTAKER_PERSONIDENTNUMBER
+import no.nav.syfo.testhelper.generator.*
+import no.nav.syfo.util.defaultZoneOffset
+import org.amshove.kluent.shouldBeFalse
+import org.amshove.kluent.shouldBeTrue
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class OppfolgingstilfelleArbeidstakerSpek : Spek({
```

Comment:
Fine tester 🏆 
Kan godt droppe "should" fra it-ene for min del, holder med: 
`it("returns false if tilfelleEnd is before dialogmotekandidatStoppunktPlanlagt, and list of DialogmoteKandidatEndring is empty")`

Reply:
Enig, endret `should return` -> `returns` nå.